app/views/modules/_module_result.html.erb

```diff
@@ -5,7 +5,7 @@
     <% else %>
       <i class="icon-file"></i>
     <% end %>
-    <%= link_to format_module_name(module_result.kalibro_module.name), "#module_#{module_result.id}", onClick: "Module.Tree.load('#{escape_javascript(image_tag 'loader.gif')} Loading data. Please, wait.', #{module_result.id});" %>
+    <%= link_to format_module_name(module_result.kalibro_module.name), "#module_#{module_result.id}", onClick: "Module.Tree.load('#{escape_javascript(image_tag 'loader.gif')} #{escape_javascript(t('loading_data'))}', #{module_result.id});" %>
```

Comment:
Do you have any javascript to escape on ```t('loading_data')```?

Or do you have HTML? If yes, maybe would be better to use ```t('loading_data').html_safe``` instead of escaping JS.